evm_body.yaml

```diff
@@ -692,6 +706,7 @@ alchemy_getTransactionReceipts_param:
                 type: string
                 description: The block hash you want to get transaction receipts for
                 pattern: '^0[xX][0-9a-fA-F]+$'
+                default: '0xb3e8c898cfbf4072eaad440e8606e578a33ca4fafc27d7936d83d7392ba3e939'
```

Comment:
this seems like a component we should reuse? and set the default on the component itself

Reply:
Will do this!